dist/index.html

```diff
@@ -33,6 +37,8 @@
         <div class="row">
           <div class="content">
             <ul>
+              <li>10/24(Sat) <a href="#GPU"><strong>今井研が所有する計算資源の情報を追加しました！</strong></a></li>
+              <li>10/24(Sat) <a href="#デモ"><strong>今井研で行われている研究についての情報を追加しました！</strong></a></li>
               <li>10/20(Tue) <a href="#全体説明会"><strong>全体説明会についての情報を追加しました！</strong></a></li>
               <li>10/19(Mon) <a href="#オンライン懇談会"><strong>オンライン懇談会についての情報を追加しました！</strong></a></li>
               <li>10/19(Mon) <a href="#forb3"><strong>B3の皆さんへを追加しました！</strong></a></li>
```

Comment:
idですが、日本語にするのはあまり見ない記法ですし、誤動作を防ぐ上でもスネークケースに統一してもらえると助かります。
（それぞれの箇所もセットで書き換えで...）

```suggestion
              <li>10/24(Sat) <a href="#gpu"><strong>今井研が所有する計算資源の情報を追加しました！</strong></a></li>
              <li>10/24(Sat) <a href="#demo"><strong>今井研で行われている研究についての情報を追加しました！</strong></a></li>
              <li>10/20(Tue) <a href="#session"><strong>全体説明会についての情報を追加しました！</strong></a></li>
              <li>10/19(Mon) <a href="#reception"><strong>オンライン懇談会についての情報を追加しました！</strong></a></li>
              <li>10/19(Mon) <a href="#forb3"><strong>B3の皆さんへを追加しました！</strong></a></li>
```

Reply:
同じ箇所ですが、スマホで見た時少し崩れてしまうことを確認しました。
（灰色の罫線の右に余白ができちゃってる = What's newの部分が右にはみ出してる）
![2020-10-25](https://user-images.githubusercontent.com/20468497/97106485-3e27c080-1705-11eb-9abf-b4a94303ca3f.png)

Reply:
tableを使ったこんな書き方はどうでしょう？
ついでに、スマホで文字サイズ20pxはちょっと大きい気がしたので小さめにしました。

html
```(html)
      <table id="whats_new">
        <tr>
          <td>10/24(Sat)</td>
          <th><a href="#gpu">今井研が所有する計算資源の情報を追加しました！</a></th>
        </tr>
        <tr>
          <td>10/24(Sat)</td>
          <th><a href="#demo">今井研で行われている研究についての情報を追加しました！</a></th>
        </tr>
        <tr>
          <td>10/20(Tue)</td>
          <th><a href="#session">全体説明会についての情報を追加しました！</a></th>
        </tr>
        <tr>
          <td>10/19(Mon)</td>
          <th><a href="#reception">オンライン懇談会についての情報を追加しました！</a></th>
        </tr>
        <tr>
          <td>10/19(Mon)</td>
          <th><a href="#forb3">B3の皆さんへを追加しました！</a></th>
        </tr>
      </table>
```

css
```
table#whats_new {
  margin: auto;
  font-size: 14px;
  border-collapse: separate;
  border-spacing: 20px 3px;
  td {
    text-align: left;
  }
}
```

スクリーンショット
|phone|PC|
|--|--|
|![2020-10-25 (1)](https://user-images.githubusercontent.com/20468497/97106697-8c898f00-1706-11eb-9445-7181199fa743.png)|![2020-10-25 (2)](https://user-images.githubusercontent.com/20468497/97106683-80053680-1706-11eb-8f25-a7f840ec3eca.png)|